docs/content/en/docs-v0.22.x/operator-manual/piped/configuration-reference.md

```diff
@@ -24,8 +24,7 @@ spec:
 | pipedKeyFile | string | The path to the file containing the generated key string for this piped. | Yes |
 | pipedKeyData | string | Base64 encoded string of Piped key. Either pipedKeyFile or pipedKeyData must be set. | Yes |
 | apiAddress | string | The address used to connect to the control-plane's API. | Yes |
-| syncInterval | duration | How often to check whether an application should be synced. Default is `1m`. | No |
-<!-- TODO: Add doc for appConfigSyncInterval -->
+| syncInterval | duration | How often to check whether an application should be synced. Default is `1m`. | No | <!-- TODO: Add doc for appConfigSyncInterval -->
```

Comment:
To fix the broken layout of the table.
https://pipecd.dev/docs-dev/operator-manual/piped/configuration-reference/#piped-configuration

Reply:
Oops. Thanks